pontoon/projects/views.py

```diff
@@ -102,7 +94,7 @@ def ajax_notifications(request, slug):
         # For performance reasons, only filter contributors for selected
         # locales if different from all project locales
         available_ids = sorted(list(available_locales.values_list('id', flat=True)))
-        selected_ids = sorted(map(int, selected_ids.split(',')))
+        selected_ids = sorted(map(int, form.cleaned_data.get('selected_locales').split(',')))
```

Comment:
I remember there's a function called 'split_ints' -> maybe we could add something similar.